crates/bevy_transform/src/helper.rs

```diff
@@ -0,0 +1,137 @@
+//! System parameter for computing up-to-date [`GlobalTransform`]s.
+
+use bevy_ecs::{
+    prelude::Entity,
+    query::QueryEntityError,
+    system::{Query, SystemParam},
+};
+use bevy_hierarchy::{HierarchyQueryExt, Parent};
+
+use crate::components::{GlobalTransform, Transform};
+
+/// System parameter for computing up-to-date [`GlobalTransform`]s.
```

Comment:
This should note the performance penalty that comes with using this SystemParam over relying on the value in a `GlobalTransform` component, and what kind of use cases it should be used.